narwhals/_polars/expr.py

```diff
@@ -60,21 +61,23 @@ def ewm_mean(
         min_periods: int,
         ignore_nulls: bool,
     ) -> Self:
-        if self._backend_version < (1,):  # pragma: no cover
-            msg = "`ewm_mean` not implemented for polars older than 1.0"
-            raise NotImplementedError(msg)
         expr = self._native_expr
-        return self._from_native_expr(
-            expr.ewm_mean(
-                com=com,
-                span=span,
-                half_life=half_life,
-                alpha=alpha,
-                adjust=adjust,
-                min_periods=min_periods,
-                ignore_nulls=ignore_nulls,
-            )
+
+        native_expr = expr.ewm_mean(
+            com=com,
+            span=span,
+            half_life=half_life,
+            alpha=alpha,
+            adjust=adjust,
+            min_periods=min_periods,
+            ignore_nulls=ignore_nulls,
         )
+        if self._backend_version < (1,):  # pragma: no cover
+            pl = get_polars()
```

Comment:
we should be ok to just import polars here in `_polars`

Reply:
Done.